daisy_workflows/image_import/import_image.sh

```diff
@@ -140,13 +140,15 @@ fi
 # Ensure the output disk has sufficient space to accept the disk image.
 # Disk image size info.
 SIZE_BYTES=$(qemu-img info --output "json" "${IMAGE_PATH}" | grep -m1 "virtual-size" | grep -o '[0-9]\+')
+IMPORT_FILE_FORMAT=$(qemu-img info "${IMAGE_PATH}" | grep -m1 "file format" | grep -oP '(?<=^file format:[ *]).*')
```

Comment:
Minor (not a regex expert): match end of line after .*

Reply:
It's not required - grep always output single line; since ".*" matches everything in a line, it is ended at end of line.